rules/opentitan_test.bzl

```diff
@@ -236,6 +236,7 @@ def fpga_params(
         tags = _BASE_PARAMS["tags"],
         test_runner = _BASE_PARAMS["test_runner"],
         test_cmds = [
+            "--exec=\"transport init\"",
```

Comment:
You probably need to update the [new rules as well](https://github.com/lowRISC/opentitan/blob/master/hw/top_earlgrey/BUILD#L55).

Reply:
I seems that Douglas beat me to it.